pkg/services/ngalert/store/alert_rule.go

```diff
@@ -157,6 +158,17 @@ func (st DBstore) DeleteRuleGroupAlertRules(orgID int64, namespaceUID string, ru
 	return ruleUIDs, err
 }
 
+// DeleteAlertInstanceByRuleUID is a handler for deleting alert instances by alert rule UID when a rule has been updated
```

Comment:
`s/DeleteAlertInstanceByRuleUID/DeleteAlertInstancesByRuleUID` (Instance -> Instances) since it can do multiple, and other Methods seem to be pluralized.